README.md

```diff
@@ -1 +1,128 @@
-# Hello-World-
\ No newline at end of file
+📗 Table of Contents
+📖 About the Project
+🛠 Built With
+Tech Stack
+Key Features
+🚀 Live Demo
+💻 Getting Started
+Setup
+Prerequisites
+Install
+Usage
+Run tests
+Deployment
+👥 Authors
+🔭 Future Features
+🤝 Contributing
+⭐️ Show your support
+🙏 Acknowledgements
+❓ FAQ (OPTIONAL)
+📝 License
+📖 Hello Microverse Project
+This is my first project in Microvese Full-Stack spftware development course.
+
+Hello Microverse is a project that sets up all the requirements in Microverse software development course projects. 
+
+🛠 Built With
+Tech Stack
+Describe the tech stack and include only the relevant sections that apply to your project.
+
+Client
+Server
+Database
+Key Features
+Describe between 1-3 key features of the application.
+
+[key_feature_1]
+[key_feature_2]
+[key_feature_3]
+(back to top)
+
+🚀 Live Demo
+Add a link to your deployed project.
+
+Live Demo Link
+(back to top)
+
+💻 Getting Started
+Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+Prerequisites
+In order to run this project you need:
+
+Setup
+Clone this repository to your desired folder:
+
+Install
+Install this project with:
+
+Usage
+To run the project, execute the following command:
+
+Run tests
+To run tests, run the following command:
+
+Deployment
+You can deploy this project using:
+
+(back to top)
+
+👥 Authors
+Mention all of the collaborators of this project.
+
+👤 Author1
+
+GitHub: @githubhandle
+Twitter: @twitterhandle
+LinkedIn: LinkedIn
+👤 Author2
+
+GitHub: @githubhandle
+Twitter: @twitterhandle
+LinkedIn: LinkedIn
```

Comment:
- [ ] Kindly add your real social links. Your GitHub profile is a digital foot print of you. please  follow this guide to make it as professional as possible